### PR TITLE
add resnet 50 v1 in tf fp32 int8

### PR DESCRIPTION
**Pull request by MarcelWilnicki**

### CODE REVIEW

**Review by jangrzybek — CHANGES_REQUESTED** [review]
Where to get those models? Please update the read-me.

**Review by jan-grzybek-ampere — APPROVED** [review]

**Review by jan-grzybek-ampere — COMMENTED** [review]
Do we have that resnet_50_v1 stored in S3?

**Comment by MarcelWilnicki** [comment]
> Do we have that resnet_50_v1 stored in S3?

yes, in ampere ai model zoo bucket

File: computer_vision/classification/resnet_50_v1/run.py [hunk 1]
```diff
@@ -18,14 +20,14 @@ def parse_args():
...
-                        type=str, choices=["fp32"], required=True,
+                        type=str, choices=["fp32", "int8"], required=True,
                         help="precision of the model provided")
     parser.add_argument("-b", "--batch_size",
                         type=int, default=1,
                         help="batch size to feed the model with")
     parser.add_argument("-f", "--framework",
                         type=str,
-                        choices=["pytorch", "ort"], required=True,
+                        choices=["tf", "pytorch", "ort"], required=True,
```
Comment (jan-grzybek-ampere) [inline computer_vision/classification/resnet_50_v1/run.py:30]: How to choose tflite? 😭

**Review by jan-grzybek-ampere — CHANGES_REQUESTED** [review]

File: computer_vision/classification/resnet_50_v1/run.py [hunk 2]
```diff
@@ -102,7 +156,26 @@ def main():
     args = parse_args()
     download_ampere_imagenet()
 
-    if args.framework == "pytorch":
+    if args.framework == "tf":
+        if args.precision == "fp32":
+            run_tf_fp32(**vars(args))
+        elif args.precision == "int8":
+            run_tflite_int8(**vars(args))
```
Comment (jan-grzybek-ampere) [inline computer_vision/classification/resnet_50_v1/run.py:163]: framework == "tf" but tflite is run

**Review by jan-grzybek-ampere — APPROVED** [review]

**Pull request merged**